docs/Technical-Documentation/remote-development.md

```diff
@@ -0,0 +1,76 @@
+# Remote Development
```

Comment:
Added remote-development documentation covering development on remotely hosted machines and containers.